code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -1,4 +1,4 @@
-var/list/hardcoded_gases = list("o2","n2","co2","plasma") //the main four gases, which were at one time hardcoded
+var/list/hardcoded_gases = list("o2","n2","co2","plasma", "freon") //the main four gases, which were at one time hardcoded
```

Comment:
meta gas list's length is based on how many entries are in each individual gas, not the amount of gases, iirc.

since BZ and Freon were added they would take the base 4 gases up to 6 total gases, which makes the number already wrong, hence why I'm certain you here, are wrong.